pkg/local_workflows/auth_workflow_test.go

```diff
@@ -80,9 +81,18 @@ func Test_auth_token(t *testing.T) {
 }
 
 func Test_autodetectAuth(t *testing.T) {
-	t.Run("oauth by default", func(t *testing.T) {
+	t.Run("in unstable versions, oauth by default", func(t *testing.T) {
```

Comment:
nitpick: `in unstable versions` vs `in preview versions` || `not in stable versions`?